deploy/charts/rook-ceph-cluster/templates/cephecblockpool.yaml

```diff
@@ -0,0 +1,41 @@
+{{- $root := . -}}
+{{- range $ecblockpool := .Values.cephECBlockPools -}}
+---
+apiVersion: ceph.rook.io/v1
```

Comment:
Similar to the storageclass-ec.yaml example in csi/rbd, you'll need two pools, one replicaed and one erasure coded

Reply:
So far only a single pool is defined. We still need two more resources to be defined as seen in [storageclass-ec.yaml](https://github.com/rook/rook/blob/master/deploy/examples/csi/rbd/storageclass-ec.yaml#L8):
1. A replicated pool
2. A storage class that uses both the replicated and EC pools mentioned, such as:
```yaml
  dataPool: <ec-pool-name>
  pool: <replicated-metadata-pool-name>
```